lib/new_relixir/plug/exception.ex

```diff
@@ -21,7 +21,6 @@ defmodule NewRelixir.Plug.Exception do
           case NewRelixir.CurrentTransaction.get() do
             {:ok, transaction} -> transaction
             {:error, _} -> NewRelixir.CurrentTransaction.set(conn.request_path)
-            _ -> nil
```

Comment:
Fixes
```elixir
lib/new_relixir/plug/exception.ex:24: 
The variable _ can never match since previous clauses completely covered the type {'error','not_found'} | {'ok',binary()}
```
The function
```elixir
NewRelixir.CurrentTransaction.get/0
```
will never return anything other than 
```elixir
{:ok, _}
``` 
or 
```elixir
{:error, _}
```